### PR TITLE
Added course in android development in hindi

### PR DESCRIPTION
**Pull request by Siddharth-coder13**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Description 
* [Android development | Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiL0kysYlfSOUgY5rNlOhUd) - Harry

### Why is this valuable (or not)?
It is a beginner-friendly course.

### How do we know it's really free?
It is available on youtube.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-hi.md [hunk 1]
```diff
@@ -8,6 +9,11 @@
 * [Python](#Python)
 
 
+### Android
+
+* [Android Development | Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiL0kysYlfSOUgY5rNlOhUd) - Harry
```
Comment (SethiPandi) [inline free-courses-hi.md:14]: Could we please use the name of the resource as displayed in the source?

```suggestion
* [Android Development Tutorials in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiL0kysYlfSOUgY5rNlOhUd) - Harry
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
I've just dropped a small comment!

**Comment by Siddharth-coder13** [comment]
Actually there are alphatical errors in some others files already merged, that' why it is creating a problem, but I fixed those files and opened a pull request, as soon as those gets merged it will automatically gets a green tick

**Comment by SethiPandi** [comment]
> Actually there are alphatical errors in some others files already merged, that' why it is creating a problem, but I fixed those files and opened a pull request, as soon as those gets merged it will automatically gets a green tick

Hey, yeah I'm aware of those problems! I can see the check failures are nothing to do with your PR.
I was only looking at your commit when I gave my comment, which is completely unrelated to the lint failures.

**Comment by Siddharth-coder13** [comment]
yeah, cool

**Comment by eshellman** [comment]
Thanks for alerting me to the broken build. The green checks don't appear on their own, though, I had to go back and re-run all the builds!

**Comment by Siddharth-coder13** [comment]
welcome

**Pull request merged**